addons/binding/org.openhab.binding.edimax/ESH-INF/i18n/edimax_en_EN.properties

```diff
@@ -0,0 +1,17 @@
+# binding
```

Comment:
I meant that English texts should be used on there locations and that you should normally only provide language files for other languages